### ISSUE TITLE
Can wkhtmltopdf support the mathematical formula in office word? Now all the symbols in the formula have become boxes

### ISSUE THREAD

**Issue by zhanghslq** [wkhtmltopdf/wkhtmltopdf#4381]
Can wkhtmltopdf support the mathematical formula in office word? Now all the symbols in the formula have become boxes

**Comment by PhilterPaper** [comment]
I haven't worked with this, but I'll offer a couple guesses. First, have you exported this formula as HTML (wkHTMLtoPDF doesn't directly convert Word documents)? It displays correctly in a browser? If that HTML document cannot be correctly converted by wkHTMLtoPDF, check what font(s) it's using and make sure you have them installed. If it's not something publicly available (i.e., is embedded in the document), you may be out of luck. Finally, it's possible that the Word export to HTML is using CSS that is too advanced for wkHTMLtoPDF.

**Comment by zhanghslq** [comment]
yeah,It works in Chrome when you convert HTML, but it works in boxes when you convert PDF,That character is also framed in the editor, but when displayed it is the correct symbol, probably because chrome is too powerful

**Comment by PhilterPaper** [comment]
If all the symbols have become 'tofus' (boxes), it sounds like your first problem is that you don't have whatever font Word uses for the symbols. Look in the HTML and see if you can figure out the font it's using, and if it's available to wkHTMLtoPDF. If you can get past that, you can think about dealing with any problems due to advanced CSS.
